### PR TITLE
Fix merge

### PR DESCRIPTION
**Pull request by AshleyLTW**
Fixing some errors created by the merge! Including missing dependencies for tests.

### CODE REVIEW

**Review by jiminbyun — APPROVED** [review]

File: web-app/src/test/java/com/google/sps/servlets/ReportServletTest.java [hunk 1]
```diff
@@ -19,26 +24,39 @@
...
-
-    Entity testReport = createReportEntity(request, response);
-
-    Assert.assertEquals(testReport.getProperty("title"), "Test");
-    Assert.assertEquals(testReport.getProperty("latitude"), 1.11);
-    Assert.assertEquals(testReport.getProperty("longitude"), 10.26);
-    Assert.assertEquals(testReport.getProperty("incidentType"), "Theft");
-    Assert.assertEquals(testReport.getProperty("description"), "Sample request for testing");
-    Assert.assertEquals(testReport.getProperty("timestamp"), 1496273400);
+    // HttpServletRequest request = mock(HttpServletRequest.class);
```
Comment (darrnshn) [inline web-app/src/test/java/com/google/sps/servlets/ReportServletTest.java]: nit: either uncomment this, or delete this test completely. Since this PR just fixes the merge, I would just uncomment this and make the test pass in another PR.

**Review by darrnshn — CHANGES_REQUESTED** [review]

**Review by darrnshn — APPROVED** [review]
